networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/core/encodings/RpcEncoding.java

```diff
@@ -13,16 +13,25 @@
 
 package tech.pegasys.teku.networking.eth2.rpc.core.encodings;
 
+import static tech.pegasys.teku.util.config.Constants.MAX_CHUNK_SIZE;
+
 import org.apache.tuweni.bytes.Bytes;
 import tech.pegasys.teku.networking.eth2.rpc.core.encodings.compression.snappy.SnappyFramedCompressor;
 import tech.pegasys.teku.ssz.SszData;
 import tech.pegasys.teku.ssz.schema.SszSchema;
 
 public interface RpcEncoding {
+  static RpcEncoding createRpcEncoding(int maxChunkSize) {
+    return new LengthPrefixedEncoding(
+        "ssz_snappy",
+        RpcPayloadEncoders.createSszEncoders(),
+        new SnappyFramedCompressor(),
+        maxChunkSize);
+  }
 
-  RpcEncoding SSZ_SNAPPY =
-      new LengthPrefixedEncoding(
-          "ssz_snappy", RpcPayloadEncoders.createSszEncoders(), new SnappyFramedCompressor());
+  static RpcEncoding createRpcEncoding() {
```

Comment:
I think I'd name these methods `sszSnappy` or `createSszSnappyEncoding`.  While it is currently the only `RpcEncoding` we support, it is a very specific type of encoding and I think it's important to make that clear.

Reply:
And actually I think I'd just inline this version of the method since it's only used in tests so that we have to be explicit about which max chunk size we use and keep the production code minimal.